be/src/olap/rowset/segment_v2/indexed_column_reader.h

```diff
@@ -103,7 +104,6 @@ class IndexedColumnReader : public MetadataAdder<IndexedColumnReader> {
     const KeyCoder* _value_key_coder = nullptr;
     uint64_t _mem_size = 0;
     bool _is_pk_index = false;
-    OlapReaderStatistics* _index_load_stats = nullptr;
```

Comment:
also should modify IndexedColumnIterator, it also saves OlapReaderStatistics* _stats = nullptr; at line 192